apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/Typography/Typography.tsx

```diff
@@ -0,0 +1,62 @@
+import { ReactElement } from 'react'
+import { TreeBlock } from '@core/journeys/ui'
+import Paper from '@mui/material/Paper'
+import Box from '@mui/material/Box'
+import TextFieldsRoundedIcon from '@mui/icons-material/TextFieldsRounded'
+import FormatAlignLeftRoundedIcon from '@mui/icons-material/FormatAlignLeftRounded'
+import captialize from 'lodash/capitalize'
+import lowerCase from 'lodash/lowerCase'
+import { GetJourneyForEdit_journey_blocks_TypographyBlock as TypographyBlock } from '../../../../../../../__generated__/GetJourneyForEdit'
+import { Attribute } from '../..'
+
+export function Typography({
+  id,
+  align,
+  color,
+  variant
+}: TreeBlock<TypographyBlock>): ReactElement {
+  return (
+    <>
+      <Attribute
+        id={`${id}-text-color`}
+        icon={
+          <Paper sx={{ borderRadius: 1000, overflow: 'hidden' }}>
+            <Box
+              data-testid="backgroundColorIcon"
+              sx={{
+                width: 25,
+                height: 25,
+                m: 1,
+                borderRadius: 1000,
+                backgroundColor: (theme) => color ?? theme.palette.text.primary
+              }}
+            />
+          </Paper>
+        }
+        name="Color"
+        value={captialize(color?.toString()) ?? 'None'}
+        description="Text Color"
+        // onClick open drawer
+      />
+      <Attribute
+        id={`${id}-font-variant`}
+        icon={<TextFieldsRoundedIcon />}
+        name="Font Variant"
+        value={
+          captialize(lowerCase(variant?.toString()).replace('h', 'header')) ??
+          'None'
```

Comment:
```suggestion
          'None'
```
Replace this with the actual default variant - find it in the add blocks pitch

Reply:
https://3.basecamp.com/3105655/buckets/23266566/messages/4514250301

Reply:
Doesn't that mean it can technically display a value but its actually null? If attributes are set correctly onCreate, the 'None' should never appear.

Reply:
<img width="491" alt="Screen Shot 2022-01-14 at 9 28 29 AM" src="https://user-images.githubusercontent.com/67619008/149404385-2527b9e0-a66c-4f23-950b-074374254e89.png">

The text alignment is what it looks like when I remove the `?? 'None'` part of it. Thoughts on just removing the null check?

Reply:
Not too sure what you mean but if the property is returning null, it means that the journey builder hasn't set it themselves and we're using the default property that's based on the theme / hard coded on the component. 

I thought onCreate would just set the required props leaving the optional ones to default. So if we get that null, we have to match it to what it displays by default..

Reply:
As discussed will change these to display default values